formapp/app.py

```diff
@@ -1436,10 +1448,10 @@ def endpoint_createaccount():
         db = sqlite3.connect(os.path.dirname(os.path.abspath(__file__)) + '/data/database.db')
         dbcmd = db.cursor()
         dbcmd.execute("INSERT INTO users (username, password, permissions, bot) VALUES (?, ?, ?, ?)", (name, passwordCrypt, sD.settings['perms']['anyone'], "False"))
+        return render_template("message.html", message = 'Account created.')
```

Comment:
The return has to come after the rest of the code is done. Here it will not be able to commit and close the database because it comes after the function is ended by return.